addon/components/nrg-modal-container/modal-wrapper/component.js

```diff
@@ -89,7 +90,7 @@ export default Component.extend(ResizeMixin, {
       if (!this.sidebar) {
         appliedClasses.push('content');
       }
-      if (!this.renderInPlace && !this.masterDetail && !this.sidebar && !this.lightbox) {
+      if (!this.renderInPlace && !this.masterDetail && !this.sidebar && !this.lightbox && this.scrolling) {
```

Comment:
If you just add it here, you don't have full controller over the scrolling state as you one.